<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[hiiiiiiiiiiiiiii]]>
    </line>
    <line>
      <![CDATA[How are u]]>
    </line>
  </reporter-output>
  <suite started-at="2023-05-26T20:24:14 IST" name="Default suite" finished-at="2023-05-26T20:24:14 IST" duration-ms="169">
    <groups>
    </groups>
    <test started-at="2023-05-26T20:24:14 IST" name="Default test" finished-at="2023-05-26T20:24:14 IST" duration-ms="169">
      <class name="testNGFlags.AllwaysRun">
        <test-method signature="f()[pri:0, instance:testNGFlags.AllwaysRun@7f552bd3]" started-at="2023-05-26T20:24:14 IST" name="f" finished-at="2023-05-26T20:24:14 IST" duration-ms="25" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[hiiiiiiiiiiiiiii]]>
            </line>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method signature="g()[pri:0, instance:testNGFlags.AllwaysRun@7f552bd3]" started-at="2023-05-26T20:24:14 IST" name="g" finished-at="2023-05-26T20:24:14 IST" duration-ms="8" status="FAIL">
          <exception class="java.lang.ArithmeticException">
            <message>
              <![CDATA[/ by zero]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArithmeticException: / by zero
at testNGFlags.AllwaysRun.g(AllwaysRun.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArithmeticException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- g -->
        <test-method signature="h()[pri:0, instance:testNGFlags.AllwaysRun@7f552bd3]" started-at="2023-05-26T20:24:14 IST" name="h" finished-at="2023-05-26T20:24:14 IST" duration-ms="6" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[How are u]]>
            </line>
          </reporter-output>
        </test-method> <!-- h -->
      </class> <!-- testNGFlags.AllwaysRun -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
